hw5/pkg/game.go

```diff
@@ -0,0 +1,153 @@
+package game
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/pkg/term"
+)
+
+// Raw input keycodes
+const (
+	up     byte = 65
+	down   byte = 66
+	left   byte = 68
+	right  byte = 67
+	escape byte = 27
+	enter  byte = 13
+)
+
+type Game struct {
+	Title   string
+	Players []Player
+	Board   Board
+}
+
+func NewGame() *Game {
+	return &Game{
+		Title: "TicTacToe, 'X' starts the game.\n" +
+			"Press Enter to make turn.\n" +
+			"Press ESC to exit.",
+		Players: make([]Player, 0),
+	}
+}
+
+func (game *Game) ConfigureGame() {
+	for index, symbol := range Players() {
+		game.Players = append(game.Players, *NewPlayer(symbol))
+		if index == 0 {
+			game.Players[0].ChangeTurn()
+		}
+	}
+	game.Board = *NewBoard(3)
```

Comment:
I think you want to store pointer to a board. Currently you are copying entire struct on assignment